node-tests/fixtures/route/strict-route-template.gts

```diff
@@ -0,0 +1,14 @@
+import type { TOC } from '@ember/component/template-only';
+import { pageTitle } from 'ember-page-title';
+
+interface FooSignature {
+  Args: {
+    model: unknown;
+    controller: unknown;
```

Comment:
I matched what the [template-tag codemod outputs](https://github.com/embroider-build/embroider/blob/4574bb78d25a3d2bffc602b1484e089b0f5c4af1/packages/template-tag-codemod/src/index.ts#L121).